static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -188,132 +155,59 @@ const pageMatchesTags = (tagIds: string[]): boolean =>
 const pageMatchesSections = (sectionIds: string[]): boolean =>
     sectionIds.some(section => config.get('page.section') === section);
 
-const makeABTestVariant = (
-    id: string,
-    products: $ReadOnlyArray<OphanProduct>,
-    test?: (html: string, abTest: ABTest) => void,
-    deploymentRules: DeploymentRules = defaultMaxViews,
-    options: Object,
+const makeEpicABTestVariant = (
+    initVariant: InitEpicABTestVariant,
     template: EpicTemplate,
     parentTest: EpicABTest
-): Variant => {
+): EpicVariant => {
     const trackingCampaignId = `epic_${parentTest.campaignId}`;
     const componentId = createTestAndVariantId(
         parentTest.campaignPrefix,
         parentTest.campaignId,
-        id
+        initVariant.id
+    );
+    const campaignCode = createTestAndVariantId(
+        parentTest.campaignPrefix,
+        parentTest.campaignId,
+        initVariant.id
     );
-    const iframeId = `${parentTest.campaignId}_iframe`;
-
-    // defaults for options
-    const {
```

Comment:
The diff here is very confusing, but essentially it now just creates an `EpicVariant` from the given `InitEpicABTestVariant` plus some default values, rather than using the `options` object